.env.example

```diff
@@ -42,3 +42,7 @@ USERS_LOGIN_QUEUE_NAME=ulqueuename
 TEST_LOGIN_FISCAL_CODES=AAABBB01C02D345Z,EEEFFF01C02H345Z
 TEST_LOGIN_PASSWORD=secret
 FF_BONUS_ENABLED=1
+JWT_SUPPORT_TOKEN_PRIVATE_RSA_KEY="-----BEGIN RSA PRIVATE KEY-----\n
```

Comment:
I guess we cannot merge & deploy this PR since these values are required until they're set up in the  production app service, isn't it?

Reply:
Yes I'll prepare another PR on infrastructure :)

Reply:
Done [https://github.com/pagopa/io-infrastructure-live-new/pull/290](https://github.com/pagopa/io-infrastructure-live-new/pull/290)

Reply:
lgtm, once applied we can merge this and deploy